source/Symbol/SwiftASTContext.cpp

```diff
@@ -3413,9 +3413,8 @@ static void
 GetLibrarySearchPaths(std::vector<std::string> &paths,
                       const swift::SearchPathOptions &search_path_opts) {
   paths.clear();
-  paths.resize(search_path_opts.LibrarySearchPaths.size() + 1);
-  std::copy(search_path_opts.LibrarySearchPaths.begin(),
-            search_path_opts.LibrarySearchPaths.end(), paths.begin());
+  paths.assign(search_path_opts.LibrarySearchPaths.begin(),
```

Comment:
why not simply
`paths = search_path_opts.LibrarySearchPaths`
?

Reply:
Dunno, that's what Shafik suggested.